visualization/app/codeCharta/ui/fileChooser/fileChooser.e2e.ts

```diff
@@ -0,0 +1,41 @@
+import { goto, newPage, launch } from "../../../puppeteer.helper"
+import { FileChooserPageObject } from "./fileChooser.po"
+import { FilePanelPageObject } from "../filePanel/filePanel.po"
+
+jest.setTimeout(15000)
+
+describe("FileChooser", () => {
+	let browser, page
+	let fileChooser: FileChooserPageObject
+	let filePanel: FilePanelPageObject
+
+	beforeAll(async () => {
+		browser = await launch()
+	})
+
+	afterAll(async () => {
+		await browser.close()
+	})
+
+	beforeEach(async () => {
+		page = await newPage(browser)
+		fileChooser = new FileChooserPageObject(page)
+		filePanel = new FilePanelPageObject(page)
+
+		await goto(page)
+	})
+
+	it("should load another cc.json", async () => {
+		await fileChooser.openFile("./app/codeCharta/assets/sample3.cc.json")
+
+		expect(await filePanel.getSelectedName()).toEqual("sample3.cc.json")
```

Comment:
This does not seem to be sufficient as test since cancelling would also end up passing here if I am not mistaken.